etl/src/main/java/org/pih/hivmigration/etl/sql/SqlMigrator.java

```diff
@@ -211,17 +219,8 @@ void loadFromOracleToMySql(String targetStatement, String sourceQuery, int rowLi
     }
 
     public void clearTable(String tableName) throws SQLException {
-        clearTable(tableName, false);
-    }
-
-    public void clearTable(String tableName, boolean disableForeignKeyChecks) throws SQLException {
-        executeMysql((disableForeignKeyChecks ? "SET FOREIGN_KEY_CHECKS=0;\n" : "SET FOREIGN_KEY_CHECKS=1;\n")
-                + "DROP TABLE IF EXISTS move_tmp_old;\n"
-                + "DROP TABLE IF EXISTS move_tmp;\n"
-                + "CREATE TABLE move_tmp LIKE " + tableName + ";\n"
-                + "RENAME TABLE " + tableName + " TO move_tmp_old, move_tmp TO " + tableName + ";\n"
-                + "DROP TABLE move_tmp_old;\n"
-                + (disableForeignKeyChecks ? "SET FOREIGN_KEY_CHECKS=1;\n" : ""));
+        executeMysql("Deleting entries from table '" + tableName + "'",
+                "SET FOREIGN_KEY_CHECKS = 0;\n TRUNCATE TABLE " + tableName + ";\n SET FOREIGN_KEY_CHECKS = 1;");
```

Comment:
I still don't like it, but I guess I can live with it (the disabling foreign key checks).  I'd rather we not do it if we can avoid it.